run_websocket_server.py

```diff
@@ -152,7 +152,8 @@ def target():
     parser.add_argument(
         "--testing",
         action="store_true",
-        help="if set, websocket server worker will load the test dataset instead of the training dataset",
+        help="""if set, websocket server worker will load
+                the test dataset instead of the training dataset""",
```

Comment:
This won't look very good because the \n and all the spaces at the beginning of the second line will be in the string.
You can do something like
```
help = (
    "beginning of loooooong sentence "
    "the end of it."
)
```

Reply:
Done